src/openms_gui/source/VISUAL/TOPPViewMenu.cpp

```diff
@@ -175,7 +180,7 @@ namespace OpenMS
     m_help->addAction("Tutorials and documentation", [&]() { GUIHelpers::openURL("html/index.html"); }, Qt::Key_F1);
 
     m_help->addSeparator();
-    m_help->addAction("&About", [&]() {QApplicationTOPP::showAboutDialog(parent, "TOPPView"); });
+    m_help->addAction("&About", [parent]() {QApplicationTOPP::showAboutDialog(parent, "TOPPView"); });
```

Comment:
add comment that this is important?

Reply:
The complete line or the keyword *parent* instead of *&*